.travis.yml

```diff
@@ -7,12 +7,25 @@ node_js:
 # dist: xenial
 # os: linux
 
+before_install:
```

Comment:
we have to do some extra setup to let travis use the newer version of postgres + postgis